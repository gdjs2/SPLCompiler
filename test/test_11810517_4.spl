int main(){
    char 123abc = 0
}