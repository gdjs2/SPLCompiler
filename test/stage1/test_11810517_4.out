Error type A at Line 2: unknown lexeme 123abc
Error type B at Line 2: Missing semicolon ';'
